### PR TITLE
Create workshop-DEI

### PR DESCRIPTION
**Pull request by dr-eric-jensen**
2nd part of blog series focusing on the Cyprus workshop at IEEE eScience 2023.

### CODE REVIEW

File: content/blog/workshop-DEI [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+**Diversity in Research Software: A Stark Reality**
+
+Workshop participants didn't mince words about the problem, pointing to the sobering truth that diversity diminishes as we zoom in from science and technology to specialized fields like research software engineering. "We're ignoring 60%-90% [of potential talent due to a lack of social inclusion], some large fractions of people that could be working in this job". This stark summary sets the stage for a critical conversation on how to broaden the research software talent pool.
+
+**Diagnosing the Diversity and Inclusion Problem in Research Software**
+
```
Comment (danielskatz) [inline content/blog/workshop-DEI:13]:
```suggestion
 
```

**Comment by danielskatz** [comment]
@karthik - any idea why the [preview](https://deploy-preview-46--urssi.netlify.app/blog/) don't have the new blog post? At first, I thought it was that the date was in the future when Eric first posted it, so I put in a slight change to force the preview to regenerate today (10/26), but this didn't fix the problem.

**Comment by karthik** [comment]
Yes. It doesn't work because you never gave the file an extension. Add `.md` and it should work.

**Comment by danielskatz** [comment]
ah, thanks - I didn't think to check that

**Comment by danielskatz** [comment]
preview: https://deploy-preview-46--urssi.netlify.app/blog/2023/10/26/diversity-and-inclusion-in-research-software-insights-from-an-international-workshop/

**Pull request merged**